apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -86,7 +86,7 @@ Page {
 
                             Component.onCompleted: {
                                 if (text && (text === searchEngineConfig.value)) {
-                                    searchEngine.currentIndex = index
+                                    model.currentIndex = index
```

Comment:
This should be changed back as it was as well.

Reply:
@rainemak  Done.